src/interfaces/IAssetManager.sol

```diff
@@ -6,5 +6,6 @@ import { IAssetManagedPair } from "src/interfaces/IAssetManagedPair.sol";
 interface IAssetManager
 {
     function getBalance(IAssetManagedPair owner, address token) external returns (uint112 tokenBalance);
+    function adjustManagement(IAssetManagedPair pair, int256 amount0Change, int256 amount1Change) external;
```

Comment:
This function is not used by the pairs right? What was the motive for adding it?

Reply:
correct, not used by the pairs. 

My thought was that we should define all the external functions called by pairs and owner. If we only need the functions pair needs, then we can remove it. Please advise, thanks

Reply:
I would probably want to preserve an interface that the pairs use and AMs **must** implement. This is different to what the owner needs which the AMs **may** implement.

Not sure how to split the different requirements out, have a good name for the two parts of the interface? Maybe:

- `IAssetManager`
- `IAssetManagerExtended`

... not the cleverest

Reply:
Agreed to remove the adjustManagement function entirely. Bots/owner can hit the concrete implementation

Reply:
okay removed